examples/oil-gas-automation/src/utils.ts

```diff
@@ -0,0 +1,6 @@
+export const getPrompt = (assetId: number) => `
```

Comment:
It should receive `data` as a parameter and work with `data.LowFlowRate.assetId`

Reply:
There should be no changes to this example in the PR, fixing that